src/MsalAuthProvider.ts

```diff
@@ -43,7 +42,11 @@ const StorageLocations: { localStorage: string; sessionStorage: string } = {
 };
 
 export abstract class MsalAuthProvider implements IAuthProvider {
-  public onAuthenticationStateChanged: (state: AuthenticationState, user?: IAccountInfo) => void;
+  public onAuthenticationStateChanged: (
+    state: AuthenticationState,
+    user?: IAccountInfo,
```

Comment:
I think we don't need to send IAccountInfo here if we already pass the AuthResponse, which contains the same object.